src/bootstrap/src/core/build_steps/test.rs

```diff
@@ -3600,13 +3599,10 @@ impl Step for TestFloatParse {
             &[],
         );
 
-        cargo_run.arg("--");
-        if builder.config.args().is_empty() {
+        cargo_run
+            .arg("--")
             // By default, exclude tests that take longer than ~1m.
-            cargo_run.arg("--skip-huge");
-        } else {
-            cargo_run.args(builder.config.args());
-        }
+            .arg("--skip-huge");
```

Comment:
cc @tgross35

Reply:
Doesn't this unconditionally pass `--skip-huge` rather than only if no other arguments are specified? In CI we always want this passed, but I need a way to not pass this in order to run exhaustive tests locally.

Reply:
I'll try to carry this PR forward since Onur is on vacation.